RubberduckBaseProject.csproj

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <Company>Rubberduck-VBA</Company>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFramework>net46</TargetFramework>
+    <LangVersion>7.2</LangVersion>
+    <SolutionDir>$(MSBuildProjectDirectory)</SolutionDir>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <RestorePackages>true</RestorePackages>
+    <!-- do not unify output paths by default -->
+    <UnifyOutputPath Condition=" '$(UnifyOutputPath)' == ''">false</UnifyOutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(AssemblyVersion)' == ''">
+    <!--
+     This assembly version specification is considered nonstandard.
+     As such builds that do not override the assembly version generate a warning.
+     -->
+    <AssemblyVersion>2.2.*</AssemblyVersion>
+    <FileVersion Condition=" '$(FileVersion)' == '' ">2.2.*</FileVersion>
+    <!-- Wildcards are not deterministic, ensure build passes when setting wildcard versions -->
+    <Deterministic>False</Deterministic>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" $(UnifyOutputPath) ">
+    <OutputPath>bin\</OutputPath>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+  </PropertyGroup>
+
+  <!-- Analyzer projects must set UnifyOutputPath to have this reference correctly -->
+  <Choose>
+    <When Condition=" '$(AssemblyName)' == 'RubberduckTests' ">
+      <ItemGroup>
+        <Analyzer Include="$(SolutionDir)\RubberduckTestsCodeAnalysis\bin\RubberduckTestsCodeAnalysis.dll"/>
```

Comment:
That does not make sense - it seems to be saying that if the project is `RubberduckTests`, to add `RubberduckTestsCodeAnalysis` as an analyzer for that project. But the `RubberduckTestsCodeAnalysis` project is a test project testing the analzyer, not an analyzer project... Or am I misunderstanding?